tests/e2e/electronic-prescription-coordinator-postman-tests.json

```diff
@@ -0,0 +1,149 @@
+					"options": {
+						"raw": {
+							"language": "json"
+						}
+					}
+				},
+				"url": {
+					"raw": "http://localhost:9000/ConvertSignatureFragments",
+					"protocol": "http",
+					"host": [
+						"localhost"
```

Comment:
Can we change this to use the {{base_url}} parameter as in the other e2e test so we can run it as part of the CI build